app/poros/business.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class Business
+  attr_reader :type,
+              :title,
+              :address,
+              :rating,
+              :site,
+              :img
+
+  def initialize(data)
+    @type = data[:categories][0][:title]
+    @title = data[:name]
+    # @description = ???
```

Comment:
We will need description, hours per the JSON contract

Reply:
We were keeping this as an initial until we meet tomorrow for stand-up. There is an issue with being able to grab a description. Thanks Erik!